lib/driver.js

```diff
@@ -244,6 +248,26 @@ class EspressoDriver extends BaseDriver {
       }
     };
 
+    let isUnzippedAppPath = false;
```

Comment:
can we move this block of code to a separate helper function to make the current one easier to parse?

Reply:
moved to a method -> 4178724c312ee669c02407d7f1b68d4ac6e12a6e